src/test/java/step1/StringTest.java

```diff
@@ -0,0 +1,53 @@
+package step1;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class StringTest {
+    @Test
+    @DisplayName("'1,2'을 ','로 split 했을 때 1과 2로 잘 분리되는지 확인")
+    void splitTwoNum() {
+        String input = "1,2";
+        String[] result = input.split(",");
+        assertThat(result).containsExactly("1", "2");
+    }
+
+    @Test
+    @DisplayName("'1'을 ','로 split 했을 때 1만을 포함하는 배열이 반환되는지 확인")
+    void splitOneNum() {
+        String input = "1";
+        String[] result = input.split(",");
```

Comment:
```suggestion
        String[] result = "1".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?